### DISCUSSION TITLE
Add `include-before-header` for LaTeX/PDF output (to create tagged/accessible PDF)

### DISCUSSION THREAD

**Discussion by TS-CUBED** [quarto-dev/quarto-cli#11116] [Feature Requests]
### Description

I need to produce tagged PDFs (which are a pain in LaTeX anyway) and so far have gotten around that by using HTML as my main format. 

But in the future I will need UA standard PDFs. LaTeX now has limited support for that, but needs inserting a `MetaData` section **before** the `\documentclass` header.

https://latex3.github.io/tagging-project/documentation/prototype-usage-instructions.html

Is this achievable somehow? I know I could create my own plugin/format, but was looking for an out-of-the-box option.

**Comment by mcanouil** [comment]
I don't know if that's something to be done in Quarto or upstream in Pandoc.
This is discussed in many issues/discussions there.

https://github.com/jgm/pandoc


Note that you can use the partials/template already: https://quarto.org/docs/journals/templates.html#latex-partials

You might even make your own format extension with specific YAML keys to fill your LaTeX document.

**Reply by cderv** [reply]
I think it should be up to Pandoc to offer a template update support this. Though we could also add support in Quarto in the meantime for better support of accessible PDF. 

@TS-CUBED right now the partial to modify is `doc-class.tex()`. See https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/pdf/pandoc/doc-class.tex - you can add what you need above, even use metadata to help set it. 

This is how we would do it when we'll support it
